nexus/types/src/inventory.rs

```diff
@@ -177,3 +200,57 @@ pub enum CabooseWhich {
     RotSlotA,
     RotSlotB,
 }
+
+/// Root of trust page contents found during a collection
+///
+/// These are normalized in the database.  Each distinct `RotPage` is assigned a
+/// uuid and shared across many possible collections that reference it.
+#[derive(Clone, Debug, Ord, Eq, PartialOrd, PartialEq)]
+pub struct RotPage {
+    pub data_base64: String,
```

Comment:
I'm a little surprised this doesn't have some more specific type but I see that the `base64` crate does just give you back a String.